### PR TITLE
password policy settings

### PR DESCRIPTION
**Pull request by schiessle**
initial version of the password policy app

### CODE REVIEW

File: templates/settings-admin.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+/** @var array $_ */
+/** @var OC_L10N $l */
+script('password_policy', 'settings-admin');
+style('password_policy', 'settings-admin');
+?>
+
+<div id="password-policy" class="section">
+	<h2 class="inlineblock"><?php p($l->t('Password Policy')); ?></h2>
+	<div id="password-policy-settings-msg" class="msg success inlineblock" style="display: none;">Saved</div>
```
Comment (LukasReschke) [inline templates/settings-admin.php]: l10n

Comment (schiessle) [inline templates/settings-admin.php]: That's just a placeholder and never displayed. The real string comes from here: https://github.com/nextcloud/password_policy/pull/1/files#diff-1376a269d6d13887451e97e2732f4dd4R30

**Comment by schiessle** [comment]
This is ready to review. In combination with this server PR you can already test it for public link passwords: https://github.com/nextcloud/server/pull/221

**Comment by LukasReschke** [comment]
Works. Let me try to enable travis here as well.

**Comment by LukasReschke** [comment]
Code looks good, works and tests are passing on Travis now as well => Merge.

**Pull request merged**